packages/eslint-plugin/utils/index.js

```diff
@@ -0,0 +1,46 @@
+const TRANSLATION_FUNCTIONS = [ '__', '_x', '_n', '_nx' ];
```

Comment:
Re: File naming:

- If we're throwing all utils into a single file, why not just call it `utils.js`. What other files would be present in this directory, if otherwise we're resorting to a "utils" miscellany.
- Alternatively, we could have more meaningful files for what we're introducing here (e.g. `constants.js`, `get-translate-strings.js`, etc)

Reply:
Hmm both would work.

Looking at #20574, which adds some more constants, separate files might make more sense.